docs/examples/app-with-probe/README.md

```diff
@@ -0,0 +1,42 @@
+# How to probe application's status
+
+In this section, I'll illustrate by an example how to declare a probe to test if an application is alive.
+
+## Prerequisite
+
+1. You can access a cluster(remotely or locally like `kind` or `minikube`)
+2. You have installed KubeVela
+
+## Steps
+
+### Check application yaml
+
+in [app-with-probe.yaml](./app-with-probe.yaml), you'll see a `livenessProbe` field in `properties`, which shows how to test if a component is alive.
+
+`path` is the health check path your web server exposed and `port` are your server is listening.
+
+In this example, we use `httpGet` method to check the application. It's a common method in web service. Besides the `httpGet` probe method, you can also change it into `exec` or `tcpSocket` method.
+
+### Apply the application
+
+Just apply the file in cluster.
+```shell
+kubectl apply -f app-with-probe.yaml
+```
+
+### Check the status
+
+Actually Applications in cluster are rendered into resources like `pod`.
+
+Try to describe pod:
+```shell
+$ kubectl get pod
+NAME                        READY   STATUS    RESTARTS   AGE
+frontend-86bc89d8f5-xgrnc   1/1     Running   0          18s
+
+$ kubectl describe pod frontend-86bc89d8f5-xgrnc
+...
+Liveness:     http-get http://:8080/ delay=0s timeout=1s period=10s #success=1 #failure=3
+...(other infomation)
+```
+You'll see pod is running without any error. If this component is unusable reported by livenessProbe, it will restart automatically.
```

Comment:
```suggestion
You'll see the pod is running without any errors. If this component is unusable reported by livenessProbe, it will restart automatically.
```

What do you mean by `If this component is unusable reported by livenessProbe`?

Reply:
Isn't that livenessProbe do in K8s? check if pod is unusable and tell other module?

Reply:
Got it. No problem with the expression.